core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -394,6 +394,14 @@ static <T> StreamMessage<T> aborted(Throwable cause) {
         return new AbortedStreamMessage<>(cause);
     }
 
+    /**
+     * Creates a new {@link StreamMessageWriter} that produces the objects to be published
+     * by {@link StreamMessage}.
+     */
```

Comment:
https://github.com/line/armeria/blob/bc48105148d652e2e48e788c94bec007882c31dc/core/src/main/java/com/linecorp/armeria/common/stream/StreamWriter.java#L31

I referred description from `StreamWriter`. If you have any better comment, please share to me~ I'll update it!

Reply:
How about:
```
/**
 * Creates a new {@link StreamMessageWriter} that publishes the objects written via
 * {@link StreamWriter#write(Object)}.
 */
```

Reply:
✅ updated, looks much better!